app/Trio.php

```diff
@@ -6,6 +6,7 @@
 
 class Trio extends Model
 {
+    protected $table = 'trios';
```

Comment:
To nie jest potrzebne, laravel automatycznie generuje sobie nazwę tabeli jako liczbę mnogą modelu